pageserver/src/metrics.rs

```diff
@@ -1245,132 +1247,105 @@ enum SmgrOpTimerState {
         #[allow(dead_code)]
         received_at: Instant,
     },
-    ParsedRoutedThrottledNowBatching {
+    ParsedRoutedNowThrottling {
```

Comment:
nit: I would name this `Throttling` and change `ParsedRoutedThrottledNowBatching` to `Batching`. That feels more consistent.